### PR TITLE
Fix: Remove ontolobrigde code and some Jquery code 

### PR DESCRIPTION
**Pull request by syphax-bouazzouni**
### Changes
* Remove ont_viewer data jquery code (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/822/commits/2a83bf3e1f4ec91f35312b1ef4c97078c63e08e6)
* Remove no existent fetch_results action code (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/822/commits/f3288e382b239f3c3ede4c587432503271ae2ddb)
* Remove ontology bridge code as also  removed in BioPortal (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/822/commits/49cd5307d9ffdb4c8127e760e4a9e99155502779)

### CODE REVIEW

**Comment by Bilelkihal** [comment]
@syphax-bouazzouni 
Here AGROVOC wasn't it supposed to open inside AgroPortal?
image

**Comment by syphax-bouazzouni** [comment]
> @syphax-bouazzouni Here AGROVOC wasn't it supposed to open inside AgroPortal? image
@Bilelkihal 
yes, except if there was an error while fetching the label in that case it fallback to external. 
and this is not the correct PR for the labels PR.

File: app/javascript/controllers/class_search_auto_complete_controller.js [hunk 1]
```diff
@@ -11,14 +11,9 @@ export default class extends OntoportalAutocompleteController {
...
-            // User performs a search
-            let search = confirm("Class could not be found.\n\nPress OK to go to the Search page or Cancel to continue browsing");
-
-            if (search) {
-                jQuery("#search_keyword").val(jQuery("#search_box").val());
-                jQuery("#search_form").submit();
-                return
-            }
+            // No result found
+            // TODO in this case show a message to redirect to the global search page
```
Comment (Bilelkihal) [inline app/javascript/controllers/class_search_auto_complete_controller.js:15]: Todo the todo

File: db/schema.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2020_09_21_120918) do
+ActiveRecord::Schema[7.0].define(version: 2023_11_30_210229) do
```
Comment (Bilelkihal) [inline db/schema.rb:13]: Where do we track these dates?

**Review by Bilelkihal — CHANGES_REQUESTED** [review]

**Comment by Bilelkihal** [comment]
@syphax-bouazzouni ping

File: app/javascript/controllers/class_search_auto_complete_controller.js (hunk 1 above)
Comment (syphax-bouazzouni) [reply app/javascript/controllers/class_search_auto_complete_controller.js:15]: the todo will be done in the future here https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/792

**Review by Bilelkihal — APPROVED** [review]

**Pull request merged**